src/ripple/app/misc/AmendmentTableImpl.cpp

```diff
@@ -100,8 +115,78 @@ class AmendmentTableImpl : public AmendmentTable
 void
 AmendmentTableImpl::addInitial ()
 {
-    // For each amendment this version supports, construct the AmendmentState object by calling
-    // addKnown. Set any vetoes or defaults. A pointer to the AmendmentState can be stashed
+    std::set<std::pair<uint256, std::string>> toAdd;
+
+    for (auto const& _ : detail::PreEnabledAmendments)
+    {
+        uint256 hash;
+        if (!hash.SetHex (_.first))
+        {
+            std::string const errorMsg =
+                (boost::format (
+                     "PreEnabledAmendments contains an invalid hash (expected "
+                     "a hex number). Value was: %1%") %
+                 _.first).str ();
+            throw std::runtime_error (errorMsg);
+        }
+
+        toAdd.emplace (hash, _.second);
+    }
+
+    {
+        // add the amendments from the config file
+        auto const& section = getConfig ().section (SECTION_AMENDMENTS);
+        int const numExpectedToks = 2;
+        for (auto const& _ : section.lines ())
+        {
+            boost::tokenizer<> tokenizer (_);
+            int numToks = 0;
+            uint256 id;
+            std::string friendlyName;
+            for (auto const& curTok : tokenizer)
+            {
+                ++numToks;
+                if (numToks > numExpectedToks)
+                {
```

Comment:
I usually do for a naked continue like this. I left them in because the comment ran over the 80 column limit, and with the comment on two lines I thought the braces were better.